cesium/writer_stream.go

```diff
@@ -336,65 +343,66 @@ type idxWriter struct {
 		highWaterMark telem.TimeStamp
 	}
 	// sampleCount is the total number of samples written to the index as if it were
-	// a single logical channel. I.E. N channels with M samples will result in a sample
+	// a single logical channel. i.e. N channels with M samples will result in a sample
 	// count of M.
 	sampleCount int64
 }
 
-func (w *idxWriter) Write(fr Frame) (Frame, error) {
+func (w *idxWriter) Write(ctx context.Context, fr Frame) (Frame, error) {
 	var (
-		l int64 = -1
-		c       = 0
+		lengthOfFrame     int64 = -1
+		channelsWrittenTo       = 0
```

Comment:
If you're going to rename this, at least make it related to a count i.e. `numChannelsWrittenTo` or `channelsWrittenToCount`

Reply:
And be consistent across the naming